torch/_torch_docs.py

```diff
@@ -445,6 +445,32 @@ def merge_dicts(*dicts):
             [ 0.7573, -3.9555, -2.8681]])
 """.format(**common_args, **tf32_notes))
 
+add_docstr(torch.sspaddmm,
+           r"""
+sspaddmm(input, mat1, mat2, *, beta=1, alpha=1, out=None) -> Tensor
+
+This function does exact same thing as :func:`torch.addmm` except the
+input and the output are sparse matrices.
+
+Args:
+    input (Tensor): a sparse matrix to be added
+    mat1 (Tensor): a sparse matrix to be multiplied
```

Comment:
"matrix multiplied", which is distinct from the term "multiplication"

Reply:
right, fixed.